src/BlazorWasmSdk/Targets/Microsoft.NET.Sdk.BlazorWebAssembly.6_0.targets

```diff
@@ -0,0 +1,569 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.Sdk.BlazorWebAssembly.targets
```

Comment:
Can we call this `Microsoft.NET.Sdk.BlazorWebAssembly.Current.targets`? It's not limited to 6.0.